src/native/minipal/utf8.c

```diff
@@ -1,2150 +1,669 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
-#include <minipal/utf8.h>
-
 #include <errno.h>
```

Comment:
```suggestion
#include <errno.h>
#include <stdbool.h>
```

This is C, so we should also include `stdbool.h` since I see `bool`.

Reply:
The reason why it is compiling in CI is because it is included via minipal/utf8.h. :)